guide/daily-spending-wallet/backup-and-recovery/landing-page.md

```diff
@@ -52,32 +74,27 @@ Two main types of information need to be backed up: the user's wallet recovery d
 
-Recent [studies](https://dl.acm.org/doi/fullHtml/10.1145/3411764.3445679) show users are often confused or unaware of where their private keys are being stored, ultimately causing inadequate risk assessment and poor storage behavior. This risk of user negligence may be avoided with transparent messaging around the scheme your wallet is using.
+### Other data is omitted
 
-> New-to-crypto users often expect a recovery mechanism that allows them to “regain access to their funds in the case that they lose their seed phrase.”
->
-> <cite>As researched by <a href="https://dl.acm.org/doi/fullHtml/10.1145/3411764.3445679">Voskobojnikov et al</a></cite>
+There are other pieces of data common among bitcoin wallet backups which this wallet specifically omits. For example, since the user must restore their funds within the same app, the [derivation path]({{'/guide/glossary/#derivation-path' | relative_url}}) will always be the same. Asking the user to write down their derivation path is a redundant step in this context. Also, a dedicated lightning node operator may want to have a ["Static Channel Backup"]({{'/guide/glossary/#static-channel-backup' | relative_url}}). But since the user's wallet is getting its liquidity from an LSP, there is no need for that in this scenario. The LSP will already have this information on their end.
+
+## How this wallet's backup works
 
-Cloud backups may be preferable in many situations. They speed up the onboarding process, are easier for newer users, and protect the user from mishandling their recovery phrase and potentially losing funds. They can also be automated to help ensure an up-to-date backup of channel state if the user loses their wallet. This type of gateway gives a beginner security without overwhelming them with unfamiliar onboarding actions.
+This wallet prioritizes cloud backup. It's quicker for the user and it also safeguards them from accidental loss of their backup, which could arguably be a higher risk than actual theft. That's a design decision for this daily spending wallet, which is designed for smaller amounts of money. Other apps may choose different threat models.
 
-However, there may be times when it is necessary to use a manual backup alternative, as it can be low tech and easily accessible.
+However, some users may not feel comfortable with cloud storage. Perhaps they have started using the wallet so much that they now have a lot of funds on the wallet, or perhaps they have been using bitcoin for longer and feel more comfortable with a manual backup on paper. Regardless of the reason, this wallet accommodates this by allowing the user to create a manual backup of their private key instead of or in addition to the auto cloud backup.
 
 ---
```

Comment:
A point we could add at the end of this is that if users begin to have a large amount of funds in this daily spending wallet it may be worth recommend them to move some bitcoin to a dedicated savings wallet (and link to the savings wallet reference design)